py/torch_tensorrt/dynamo/utils.py

```diff
@@ -63,6 +63,35 @@ def cosine_similarity(gt_tensor: torch.Tensor, pred_tensor: torch.Tensor) -> flo
     return res
 
 
+def input_is_dynamic(inputs: Sequence[Input]) -> bool:
```

Comment:
Should the schema be switched to `Sequence[Union[Input, torch.Tensor]]`, given the below check?

Reply:
Yes, done.